doc/source/how-to/contributing.rst

```diff
@@ -0,0 +1,250 @@
+Contributing
+============
+This page explains how PyAnsys development is conducted. When contributing to a
+PyAnsys repository, use these general coding paradigms:
+
+#. Follow the `Zen of Python <https://www.python.org/dev/peps/pep-0020/>`__.
+   As silly as core Python developers are sometimes, there's much to be
+   gained by following the basic guidelines listed in PEP 20. As suggested
+   in these guidelines, focus on making your additions intuitive, novel,
+   and helpful for PyAnsys users. When in doubt, use ``import this``.
+   For Ansys code quality standards, see :ref:`Coding Style`.
+
+#. Document your contributions. Include a docstring for any added function,
+   method, or class, following :ref:`Numpydoc Docstrings`, as specified by
+   PyAnsys :ref:`Documentation Style`. Always provide at least one simple use
+   case for a new feature.
+
+#. Test your contribution. Because Python is an interpreted language, if
+   it's not tested, it's probably broken. At the minimum, include a unit
+   test for each new feature within the ``tests`` directory. Ensure that
+   each new method, class, or function has a reasonable (>80%) coverage.
+   See :ref:`Testing` for automating testing.
```

Comment:
```suggestion
   For information about automated testing, see :ref:`Testing`.
```

Reply:
There is nothing wrong with the way line 22 was written. It's just the Ansys style to say why you'd go to the link before providing the link.